tests/e2e-playwright/Makefile

```diff
@@ -106,6 +106,29 @@ test-sleepers-dev: _check_venv_active ## runs sleepers test on local deploy
 		--autoregister \
 		$(CURDIR)/tests/sleepers/test_sleepers.py
 
+.PHONY: test-platform
+test-platform: _check_venv_active
+	@pytest \
+		-sxvv \
+		--color=yes \
+		--pdb \
```

Comment:
--pdb is to fall back in the debugger. I am not sure that is what you want when it runs in the CI.

Reply:
Be cautious—it might end up getting stuck in the CI pipeline!